app/data/models/frn-closed.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (sequelize, DataTypes) => {
+  const frnClosed = sequelize.define('frnClosed', {
+    frn: { type: DataTypes.BIGINT, primaryKey: true, autoIncrement: false }
```

Comment:
Also are we making this a generic closure capability?  If so then maybe include the schemeId it applies to in the table.  If not, then maybe name this `sfiClosures` or something so it's clear

Reply:
If we're making it generic, what if we add the `schemeId` in there too?  Then we can just retrieve all closures for a particular scheme.  Otherwise there's no way of knowing which scheme this is for.

We can just amend the upload data to include `1` as the id for SFI

Reply:
Woops - missed schemeId off my list. It is in the file though: 
![image](https://github.com/DEFRA/ffc-pay-processing/assets/60177449/7319d06c-f274-4b60-99c2-36f0aabb9f72)
And DB:
![image](https://github.com/DEFRA/ffc-pay-processing/assets/60177449/cb11d607-b255-4f77-9f00-557ae5509a8e)

Reply:
Also spotted the tagDatabase is wrong though which I've just fixed!